File: Pharmacy_project/project_BE/project_crud/project_crud/src/main/java/com/project_crud/service/PharmacistService.java
package com.project_crud.service;

import java.util.List;

import com.project_crud.model.Pharmacist;

public interface PharmacistService {

    // Add Pharmacist
    public Pharmacist addPharmacist(Pharmacist pharmacist);

    // View Pharmacists
    public List<Pharmacist> viewPharmacists();

    // Delete Pharmacist
    public String deletePharmacist(int id);
}

File: Pharmacy_project/frontend_proj/src/services/pharmacistService.js
import api from './api';

export const addPharmacist = (data) => api.post('/addPharmacist', data);
export const viewPharmacists = () => api.get('/viewPharmacists');
export const deletePharmacist = (id) => api.delete(`/deletePharmacist/${id}`);
